osu.Game.Rulesets.Mania/UI/DrawableManiaRuleset.cs

```diff
@@ -48,6 +50,10 @@ public class DrawableManiaRuleset : DrawableScrollingRuleset<ManiaHitObject>
         protected new ManiaRulesetConfigManager Config => (ManiaRulesetConfigManager)base.Config;
 
         private readonly Bindable<ManiaScrollingDirection> configDirection = new Bindable<ManiaScrollingDirection>();
+        private readonly Bindable<double> configTimeRange = new Bindable<double>();
```

Comment:
```suggestion
        private readonly Bindable<double> configTimeRange = new BindableDouble();
```
The non-generic is a tad unsafe as the underlying setting has a [`Precision` of 1](https://github.com/ppy/osu/blob/e012af661658b6569297435293af1658d381ea29/osu.Game.Rulesets.Mania/Configuration/ManiaRulesetConfigManager.cs#L23).